portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java

```diff
@@ -0,0 +1,45 @@
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns all quotes */
+@WebServlet("/quotes")
+public final class QuotesServlet extends HttpServlet {
+
+    private List<String> quotes;
+
+    @Override
+    public void init() {
+        quotes = new ArrayList<>();
+        quotes.add("When you are backed against the wall, break the goddamn thing down.");
+        quotes.add("I don’t play the odds I play the man.");
+        quotes.add("I don’t play the odds I play the man.");
+        quotes.add("You always have a choice.");
+        quotes.add("They think you care, they’ll walk all over you.");
+        quotes.add("Sorry, I can’t hear you over the sound of how awesome I am.");
+        quotes.add("Anyone can do my job, but no one can be me.");
+        quotes.add("I don’t have dreams, I have goals.");
+        quotes.add("It’s going to happen, because I am going to make it happen.");
+        quotes.add("It’s not bragging if it’s true.");
+        quotes.add("Win a no win situation by rewriting the rules.");
+        quotes.add("Let them hate, just make sure they spell your name right.");
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        String quotesJsonString = new Gson().toJson(quotes);
```

Comment:
These quotes are simple strings. Why do you want to use json?

Reply:
It's supposed to be an array of strings. I didn't want to process too much on client side. 

Are there any easier way to do this? text/plain or text/html requires me too process it using a separator may be.

PS. Also merged the next step of JSON in the walkthrough.

Reply:
Yes, thanks for merging the JSON part here.
I meant could be better if you find a better suitable use case for json. For example, in the walkthrough, you'll see a parent field people in the json, and each element under it has a name and birthyear.
Maybe we can think of a good use-case for our example too, where we can use some key-value structures and jsonify that.

Reply:
Sure. On it.

Reply:
Hey @arijit-google-sps @arijit-google-sps 

Re-requesting review.